src/apps/dashboard/dashboard-notification-list/dashboard-notification-list.tsx

```diff
@@ -292,6 +292,12 @@ const DashboardNotificationList: FC<DashboardNotificationListProps> = ({
     setAccepted(false);
   };
 
+  const isReservationNotReadyForLoan =
+    reservationForModal &&
+    readyToLoanReservations.some(
+      ({ faust }) => faust !== reservationForModal.faust
+    );
```

Comment:
Please reconsider this logic. If you want to determine whether a reservation is ready for pickup, you should check the state, like we do for `getReadyForPickup()`.

You could even use this function with a single reservation by passing the reservation within an array and checking the length of the returned array.